features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/data/api.js

```diff
@@ -505,8 +505,55 @@ class API extends Resource {
     }
 
     /**
-     * Mock api call with delay
-     * todo: Replace with the rest api when available
+     * Get list of emails configured at Bot Detection --> Configure Emails
+     */
+    botDetectionNotifyingEmailsGet() {
+        return this.client.then((client) => {
+            return client.apis['default'].get_botData_getEmailList(
+                this._requestMetaData(),
+            );
+        });
+    }
+
+    addBotDetectionNotifyingEmail(email, callback = null) {
+        const promiseAddEmail = this.client.then((client) => {
+            const data = {
+                email: email,
+            };
+            const payload = {
+                body: data,
+                'Content-Type': 'application/json',
+            };
+            return client.apis['default'].post_botData_addEmail(
+                payload,
+                this._requestMetaData(),
+            );
+        });
+
+        if (callback) {
+            return promiseAddEmail.then(callback);
+        } else {
+            return promiseAddEmail;
+        }
+    }
+
+    deleteBotDetectionNotifyingEmail(id, callback = null) {
+        const promiseDeleteEmail = this.client.then((client) => {
+            return client.apis['default'].delete_botData_deleteEmail(
+                { uuid: id },
+                this._requestMetaData(),
+            );
+        });
+
+        if (callback) {
+            return promiseDeleteEmail.then(callback);
+        } else {
```

Comment:
Same here, Not required

Reply:
Fixed all occurrences with commit: https://github.com/wso2/carbon-apimgt/pull/8591/commits/5138aa2a7e1b1611004d5260e67730dc11da6bb8